### ISSUE TITLE
Pipeline for joining with Census data

### ISSUE THREAD

**Issue by kbmorales** [JohnMcCambridge/CARES#13]
It'd be great to have a pipe script that adds on latest census data by zip code (or even congressional district 'CD')

**Comment by JohnMcCambridge** [comment]
Agreed. As of now I'm having difficulty finding open-access ZIP code level census data.

**Comment by rmcarder** [comment]
Adding ZIP (zctas), county, state, and congressional district demographics from ACS 5 year estimates.
